### ISSUE TITLE
Max OS X Panther theme request

### ISSUE THREAD

**Issue by eggjoystick** [B00merang-Project/Mac-OS-X-Cheetah#1]
Thank you SO MUCH for creating this theme. Love the B00merang Project's work. Since the unfortunate takeover of flat design and the advent of iOS 7, I dearly miss the Mac Aqua design language and have found refuge in this Cheetah theme. Any possibility of a Panther theme with the brushed metal design elements?

**Comment by Elbullazul** [comment]
you mean something like this?

![metal](https://cdn.arstechnica.net/wp-content/uploads/2012/12/finder-metal.jpg)

it's doable, and I've been wanting to try to do it for a while, but was busy with other stuff. I'll do my best to get started on this though

**Comment by eggjoystick** [comment]
Yes, exactly like your screenshot. Thanks for replying. I am not a coder or graphic design professional, but love to dabble in Inkscape, GIMP etc. If I can be of any help, please let me know. Would love to learn from you.

**Comment by eggjoystick** [comment]
Just wanted to put this here in case it is helpful for future reference. 

There is a really well done Kvantum theme created by the artist [vicko](https://www.pling.com/u/vicko/) that mimics OS X Panther which I have used personally on Manjaro 20.0.1. It is called [K**a**vantum Baghira Aqua Metal](https://www.pling.com/p/1189795/). There is also a matching Aurorae theme called [Kwin Baghira Aqua Metal Aurorae](https://www.pling.com/p/1189794/) that goes along with it.
